app/controllers/verify_controller.rb

```diff
@@ -79,6 +79,8 @@ def generate_personal_key
   def completion_url
     if session[:sp]
       sign_up_completed_url
+    elsif idv_session.address_verification_mechanism == 'gpo'
```

Comment:
This is not a viable long-term implementation, since as we start implementing more steps, won't know server-side whether the user opted for GPO verification, and instead need to determine this client-side. The feature specs should help keep us honest to implement this later, though.

Reply:
> This is not a viable long-term implementation, since as we start implementing more steps, won't know server-side whether the user opted for GPO verification, and instead need to determine this client-side. The feature specs should help keep us honest to implement this later, though.

Long-term solution proposed at #6433